tests/test_enterprise/api/test_views.py

```diff
@@ -1688,23 +1688,58 @@ def test_create_catalog(self, is_staff, expected_status_code):
         Test that a catalog can be created
         """
         enterprise_customer = factories.EnterpriseCustomerFactory(uuid=FAKE_UUIDS[0])
+        enterprise_catalog_query = factories.EnterpriseCatalogQueryFactory(uuid=FAKE_UUIDS[0])
```

Comment:
should the test catalog query really have the same UUID as the test customer?

Reply:
Ahh looking at how the code is generated I guess they are not unique on declarations. Will increment the array 👍🏽